server/utils/headers.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+import "net/http"
+
+func HandleCors(w *http.ResponseWriter, r *http.Request, allowedMethods string) {
+	(*w).Header().Set("Access-Control-Allow-Origin", "*")
+	(*w).Header().Set("Access-Control-Allow-Methods", allowedMethods)
+	(*w).Header().Set("Access-Control-Allow-Headers", "Content-Type")
+	(*w).Header().Set("Access-Control-Allow-Headers", "Accept, Content-Type, Content-Length, Accept-Encoding")
```

Comment:
Line 8 and 9 are the setting different values for the same key, they should be merged.